x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/form_errors.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { cloneElement, Children, Fragment, ReactElement } from 'react';
+import { EuiFormRow, EuiFormRowProps } from '@elastic/eui';
+
+type Props = EuiFormRowProps & {
+  errorKey: string;
+  isShowingErrors: boolean;
+  errors: Record<string, string[]>;
+};
+
+export const ErrableFormRow: React.FunctionComponent<Props> = ({
+  errorKey,
+  isShowingErrors,
+  errors,
+  children,
+  ...rest
+}) => {
+  return (
+    <EuiFormRow
+      isInvalid={isShowingErrors && errors[errorKey].length > 0}
+      error={errors[errorKey]}
+      {...rest}
+    >
+      <Fragment>
```

Comment:
It looks like this code is pre-existing, but I wonder if this `Fragment` is necessary 🤔, perhaps `EuiFormRow` requires one child.

Reply:
I think the problem is that Children.map can return an `undefined` and then `EuiFormRow` would complain